check.go

```diff
@@ -125,12 +125,19 @@ func RepairEverything() *RepairOptions {
 // Check returns a list of problems with what's in the store, as a whole.  It can be very expensive
 // to call.
 func (s *store) Check(options *CheckOptions) (CheckReport, error) {
-	var ignoreChownErrors bool
+	var ignoreChownErrors, ignoreTimestamps, ignorePermissions bool
 	for _, o := range s.graphOptions {
 		if strings.Contains(o, "ignore_chown_errors") {
 			ignoreChownErrors = true
 		}
 	}
+	for o := range s.pullOptions {
+		if strings.Contains(o, "ostree_repos") || strings.Contains(o, "use_hard_links") {
```

Comment:
The other caller consumes `use_hard_links` via `parseBooleanPullOption` (apparently allowing `"false"`), not just a presence check.

---

(It seems tempting to suggest that the individual `store.pullOptions` values should somehow only be accessed via dedicated accessors, to make the primary consumers and `check.go` clearly connected in the call graph… except that that would require a `c/storage/internal/$something` and probably isn’t worth it for the three options that currently exist.)